STPullToRefresh/STPullToRefreshHelper.h

```diff
@@ -42,9 +44,13 @@ typedef enum STPullToRefreshState {
 - (id)initWithDirection:(STPullToRefreshDirection)direction delegate:(id<STPullToRefreshHelperDelegate>)delegate;
 - (id)initWithDirection:(STPullToRefreshDirection)direction viewClass:(Class)viewClass delegate:(id<STPullToRefreshHelperDelegate>)delegate;
 
+@property (nonatomic) BOOL shouldKeepPullToRefreshViewAtFront;
+@property (nonatomic) CGFloat pullToTriggerHeightMultiplier;
```

Comment:
Is a `multiplier` the correct interface here or an `extraDistance`?
/cc @alexdeem